media/livestream/api/LiveStream.Samples/DeleteInputAsync.cs

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START livestream_delete_input]
+
+using Google.Cloud.Video.LiveStream.V1;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+
+public class DeleteInputSample
+{
+    public async void DeleteInputAsync(
```

Comment:
An async method should never be void, else, you don't give the caller the opportunity to await on the result themsevels.
```suggestion
    public async Task DeleteInputAsync(
```

Reply:
Done